tests/scripts/t3000/run_t3000_model_perf_tests.sh

```diff
@@ -66,7 +66,7 @@ run_t3000_llm_tests() {
   run_t3000_mixtral_tests
 
   # Run llama2-70b tests
-  run_t3000_llama2_70b_tests
+  # run_t3000_llama2_70b_tests
```

Comment:
why is the test removed?

Reply:
It's only temporary commented for model perf test to hit falcon40b. That is the easiest way to hit models that are not executing due to other models failing at the moment. Before merging all the changes inside tests are discarded and tests remain the same. Also, llama2 team is informed about failure on CI.

Reply:
please revert back and recomment these tests.
Llama tests are now split so this type of change shouldn't be required to test